tests/test_resources.py

```diff
@@ -244,6 +245,16 @@
         resource_type='dataflow.googleapis.com/Job',
         name='//dataflow.googleapis.com/projects/my_project/locations/us-central1/jobs/my_resource'
     ),
+    ResourceTestCase(
+        resource_data={
+            'name': test_resource_name,
+            'location': 'us-central1',
+            'project_id': test_project
+        },
+        cls=GcpMemorystoreRedis,
+        resource_type='redis.googleapis.com/Instance',
+        name='//redis.googleapis.com/projects/my_project/locations/us-central1/instances/my_resource'
+    ),
```

Comment:
We should also verify if the project_number is used, or the project_id

Reply:
confirmed